test/e2e/runner/perturb.go

```diff
@@ -28,7 +29,7 @@ func Perturb(ctx context.Context, testnet *e2e.Testnet) error {
 // after recovering.
 func PerturbNode(ctx context.Context, node *e2e.Node, perturbation e2e.Perturbation) (*rpctypes.ResultStatus, error) {
 	testnet := node.Testnet
-	out, err := execComposeOutput(testnet.Dir, "ps", "-q", node.Name)
+	out, err := docker.ExecComposeOutput(context.Background(), testnet.Dir, "ps", "-q", node.Name)
```

Comment:
will this eventually work also for DO?

Reply:
I think the point of the PR was just to start putting in some of the initial plumbing to get the DO nodes to start/stop. The original intention was to expand this functionality with perturbations in follow-up PRs.

Reply:
To add on to @thanethomson's comment, `PerturbNode` will eventually become a method in `infra.Provider`, whose Docker version will be + or - it current implementation; and we will have to come up with mechanisms for implementing it for a DO testnet.